pkg/config/config_test.go

```diff
@@ -34,27 +31,15 @@ var testConfigFilePath = os.Getenv("HOME") + "/.simple-ec2/" + testConfigFileNam
 func TestSaveInConfigFolder(t *testing.T) {
 	testString := "unit test config"
 	testData := []byte(testString)
-	path, err := config.SaveInConfigFolder(testConfigFileName, testData, 0644)
-	if err != nil {
-		t.Fatal(err)
-	}
 
-	// Check if the path is correct
-	if *path != testConfigFilePath {
-		os.Remove(*path)
```

Comment:
is this required? I am assuming testHelper won't be running `os.Remove`.

Also, would we loose some of the clear messaging with these changes? E.g. `The config file path is incorrect` is clear when the user sees the error as opposed to something like these 2 values don't match.

Reply:
* Good call, this is most deff required. I'll update in next revision
* Truee, I'll update with `assert` instead and keep original messaging

Reply:
On the larger scale, I think foregoing the exact messaging is worth it so we can use these helpers and make the tests clear+readable. Unit test should be isolated enough so if it does fail, then whomever needs to fix it just needs to know what was expected and can deep dive from there.